packages/react-native-cli/src/lib/Gradle.ts

```diff
@@ -268,17 +249,22 @@ See ${DOCS_LINK} for more information`
   }
 }
 
-async function insertValueAfterPattern (file: string, pattern: RegExp, value: string, presencePattern: RegExp, logger: Logger): Promise<void> {
+async function insertValueAfterPattern (file: string, pattern: RegExp, pattern2: RegExp, value: string, presencePattern: RegExp, logger: Logger): Promise<void> {
   const fileContents = await fs.readFile(file, 'utf8')
 
   if (presencePattern.test(fileContents)) {
     logger.warn('Value already found in file, skipping.')
     return
   }
-
-  const match = fileContents.match(pattern)
+  let match = fileContents.match(pattern)
```

Comment:
with the above array suggestion:

```suggestion
  let match = patterns.find(search => fileContents.match(search))
```